helm/conjur-app-deploy/charts/app-secrets-provider-init/templates/NOTES.txt

```diff
@@ -0,0 +1,8 @@
+The Application deployment is complete.
+The following have been deployed:
+{{ if .Values.conjur.authnConfigMap.create }}
+- A Conjur authentication configmap
+{{ end }}
+{{ if .Values.create }}
```

Comment:
I think it would also be useful to have information about the internal endpoint where this application will be available, i.e.
```
test-app-backend.$TEST_APP_NAMESPACE_NAME.svc.cluster.local
```

Reply:
Removed the conditional and added the connection URL to NOTES.txt